File: 241022/물의 상태/state-of-water.cpp
#include <iostream>
using namespace std;

int main() {
    int temperature;
    cin >> temperature;

    if (temperature < 0) {
        cout << "ice";
    } else if (temperature >= 100) {
        cout << "vapor";
    } else {
        cout << "water";
    }
    return 0;
}

File: 241022/물의 상태/state-of-water.js
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim();
let temperature = Number(input);

if (temperature < 0) {
    console.log("ice");
} else if (temperature >= 100) {
    console.log("vapor");
} else {
    console.log("water");
}